spec/03-plugins/14-request-termination/01-schema_spec.lua

```diff
@@ -1,5 +1,6 @@
 local schema_def = require "kong.plugins.request-termination.schema"
 local v = require("spec.helpers").validate_plugin_config_schema
+local null = ngx.null
```

Comment:
It seems that this localization is unnecessary.

Reply:
It will be used in this line https://github.com/Kong/kong/blob/0ee88ee3492f03a7f47d3d2f615eae3dae5ee461/spec/03-plugins/14-request-termination/01-schema_spec.lua#L29

Reply:
Sorry, my meaning is that we can use `ngx.null` in this line, need not to localize it.

Reply:
I don't think it matters as it seems like a trivial style.🙂